For R.C. - 20404: Edit packages feature (#22169)
fleetdm/fleet: docs/Contributing/Audit-logs.md

}
```

## edited_software

Generated when a software installer is updated in Fleet.

This activity contains the following fields:
- "software_title": Name of the software.

> - "software_package": Filename of the installer. `null` if the installer package was not modified.

Reviewer: @iansltx updated this description in https://github.com/fleetdm/fleet/pull/22177/files earlier today, so I think to avoid conflicts it should now be:
```suggestion
- "software_package": Filename of the installer as of this update (including if unchanged).
```
